lib/Byng/Pimcore/Sitemap/Generator/SitemapGenerator.php

```diff
@@ -36,26 +36,30 @@
     /**
      * @var SimpleXMLElement
```

Comment:
I mean the dockblock here is for the `private $host` field and not for the `private $xml` field which is further down.
`

    /**
     * @var SimpleXMLElement
     */
    private $host;
    /**
     * @var SimpleXMLElement
     */
    private $xml;

`